net/isc-dhcp/Makefile

```diff
@@ -18,7 +18,7 @@ PKG_MAINTAINER:=Antony Antony <antony@phenome.org>
 
 PKG_SOURCE:=$(UPSTREAM_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=ftp://ftp.isc.org/isc/dhcp/$(PKG_VERSION)
```

Comment:
Please add at least two http mirrors and use isc as fallback (last)

http://ftp.funet.fi/pub/mirrors/ftp.isc.org/isc/dhcp/4.3.5/
http://ftp.ntua.gr/pub/net/isc/isc/dhcp/4.3.5/
http://ftp.iij.ad.jp/pub/network/isc/dhcp/4.3.5/